media-libs/libcuefile/libcuefile-477-r1.ebuild

```diff
@@ -22,4 +22,5 @@ src_install() {
 	cmake-multilib_src_install
 	insinto /usr/include
 	doins -r include/cuetools
+	find "${D}" -type f -name '*.a' -delete || die
```

Comment:
no way to disable this in cmake? :cry:

Reply:
Unfortunately not with configure options :/
I can add this patch, if it is more preferable than deleting static libs at install time
```diff
From 92d3205dac3c93937436ebc4666d9091de405f87 Mon Sep 17 00:00:00 2001
From: Jakov Smolic <jakov.smolic@sartura.hr>
Date: Sun, 20 Sep 2020 14:34:47 +0200
Subject: [PATCH] src: CMakeLists: disable static libs

Signed-off-by: Jakov Smolic <jakov.smolic@sartura.hr>
---
 src/CMakeLists.txt | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c07fe0d..1bac1aa 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,10 +1,7 @@
 add_definitions(-DYY_NEVER_INTERACTIVE)
 include_directories(${libcuefile_SOURCE_DIR}/include)
 
-add_library(cuefile-static STATIC cd cdtext cue_parse cue_print cue_scan cuefile time toc toc_parse toc_print toc_scan)
-set_target_properties(cuefile-static PROPERTIES OUTPUT_NAME cuefile CLEAN_DIRECT_OUTPUT 1)
-
 add_library(cuefile-shared SHARED cd cdtext cue_parse cue_print cue_scan cuefile time toc toc_parse toc_print toc_scan)
 set_target_properties(cuefile-shared PROPERTIES OUTPUT_NAME cuefile CLEAN_DIRECT_OUTPUT 1 VERSION 0.0.0 SOVERSION 0)
 
-install(TARGETS cuefile-static cuefile-shared LIBRARY DESTINATION "lib${LIB_SUFFIX}" ARCHIVE DESTINATION "lib${LIB_SUFFIX}")
+install(TARGETS cuefile-shared LIBRARY DESTINATION "lib${LIB_SUFFIX}" ARCHIVE DESTINATION "lib${LIB_SUFFIX}")
-- 
2.26.2
```

Reply:
Yes please, could you try to upstream it? The perfect solution would not specify `SHARED`/`STATIC` in `add_library` but just rely on `BUILD_SHARED_LIBS` (which is enabled since EAPI 7).

Reply:
Ok, no problem, but these projects seem to be dead upstream, last releases in 2011 as far as I can see

Reply:
All right, I added this simple patch for 2 ebuilds. Most of the other packages require simple `--disable-static` in econf, but some packages do not have the configure option to disable static, and do not seem to be trivial to patch, so I just removed `.a` files in those, I hope that's okay. Let me know if I need to fix anything else.

btw, I see huge amount of these static-libs bugs, I will work on fixing those over the next few days, is it okay if I ping you ocasionally to merge PRs?\
Thanks a lot! :)

Reply:
@jsmolic that's fine, great work by the way!

Reply:
and yes, only bother upstreaming stuff for non-dead projects obviously

Reply:
Thanks!
Alright, I did some more, I think this could be it for the first batch. All bugs seem to be correctly linked, so this can be merged ;)
I'll do another batch by the end of the week.

p.s. I didn't blindly add `econf --disable-static` everywhere. I peaked into the sources, and compiled the ebuilds afterwards to make sure I didn't miss something out